dev/modules/utils.js

```diff
@@ -61,11 +61,56 @@ var colorLuminance = function(hex, lum) {
   return rgb;
 };
 
+var body = document.getElementsByTagName('body')[0];
```

Comment:
This code don't work if JS is loaded in HEAD, not in end of boudy.
Should use document.ready or something like that.